package.json

```diff
@@ -44,7 +44,7 @@
       "batch-print": "1.0",
       "circulation": "9.0 10.0 11.0 12.0 13.0 14.0",
       "consortia": "1.0",
-      "feesfines": "16.1 17.0 18.0 19.0",
+      "feesfines": "16.1 17.0 18.0 19.0 19.1",
```

Comment:
@zburke 19.0 exists already. Should 19.1 explicitly added here?

Reply:
It depends :) `"feesfines": "16.1 17.0 18.0 19.0 19.1"` means "this module can run just fine when _any_ of these interfaces are present". In this context, `19.1` is redundant to `19.0` because the semver equivalent of `19.0` is `^19`. 

But that's not the most important question, which is, "Given this PR removes many permissions from existing permission-sets, what will the impact be?" e.g. the user-assignable pset `ui-users.feesfines.actions.all` has `accounts.transfer.post` replaced with `feesfines.accounts.transfer.item.post`. What would happen in an environment where only `19.0` (or only `16.1`!) is provided? Will the user be able to do the same things before and after this change? I think the answer is "No" because this PR removes `accounts.transfer.post` (even though exists on the backend in `v19.0`) and replaces it with `feesfines.accounts.transfer.item.post` (which does NOT exist on the backend in `v19.0`). IOW, removing permissions from existing psets is a breaking change. 

There are two options:
1. Accept the breaking change: Replace old permissions with new ones as the PR currently does, AND bump the minimum interfaces (e.g. `"feesfines": "19.1"`), AND bump the major version of this repository
2. Do this work in a backwards-compatible way: Add new permissions along-side the old permissions, i.e. instead of  replacing `accounts.transfer.post` with `feesfines.accounts.transfer.item.post`, include them both in the same pset. I believe it's possible to do this, and mod-permissions will simply ignore the permissions that don't exist (i.e. in a `19.0` env it will ignore the new perm that doesn't exist (because it has not been created yet), and in a `19.1` env it will ignore the old perm that doesn't exist (because it has already been removed)). Please confirm this behavior with somebody who works on mod-permissions, maybe @julianladisch? He is tech-lead on that module. 

Finally, please confirm that these new permissions are all correct. The one I happened to pick as an example, `feesfines.accounts.transfer.item.post`, does not appear in either of the two linked PRs.